prow/cluster/jobs/istio/istio/istio.istio.master.yaml

```diff
@@ -218,11 +218,10 @@ presubmits:
           - entrypoint
           - prow/e2e-simpleTests-cni.sh
         nodeSelector:
-    - name: build-tests
+    - name: release-test
```

Comment:
move this to postsubmit for now.

failure like the one we had recently is just noise to be in presubmit. also you should set up testgrid to see historical results.